index.css

```diff
@@ -0,0 +1,540 @@
+  font-family: 'Lato', sans-serif;
+  font-weight: 700;
+  font-size: 0.6rem;
+  line-height: 1rem;
+  color: #000;
+  top: 0;
+  margin-bottom: 0.2rem;
+}
+
+.speaker-details .info hr {
+  border: 1.5px solid #d9d9d9;
+}
+
+.see-more {
+  display: flex;
+  width: 95%;
+  padding: 0.5rem 2rem;
+  align-items: center;
+  justify-content: center;
+  gap: 0.2rem;
+  border: 1px solid #e0e0e0cd;
+  background: #fff;
+  box-shadow: 4px 4px 4px rgba(0, 0, 0, 0.25);
+  border-radius: 0.5rem;
+  font-family: 'Lato', sans-serif;
+  font-weight: 700;
+  font-size: 0.7rem;
+  color: #000;
+}
+
+.see-more i {
+  color: #ff4f38;
+}
+
+footer,
+.join-btn2,
+.partner {
+  display: none;
+}
+
+/* Media Query for desktop Version  */
+
+@media only screen and (min-width: 768px) {
+  header {
+    position: relative;
+  }
+
+  nav .top {
+    display: flex;
+    width: 100%;
+    align-items: center;
+    justify-content: flex-end;
+    margin: 0 auto;
+    background: #3e3e3e;
+    padding: 0.2rem 1rem;
+    gap: 1rem;
+  }
+
+  nav .top a {
+    font-family: 'Lato', sans-serif;
+    color: #fff;
+  }
+
+  nav .bottom {
+    width: 90%;
+    display: flex;
+    padding: 0.5rem;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+  }
+
+  nav .bottom .nav-menu {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: 1rem;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .nav-menu li:nth-child(6) {
+    padding: 0.5rem;
+    border: 5px solid #ec4327;
+  }
+
+  .nav-menu li:nth-child(6) a {
+    padding: 0.5rem;
+    color: #ec4327;
+  }
+
+  .nav-link {
+    font-family: 'Lato', sans-serif;
+    color: #868686;
+  }
+
+  .hero-wrapper h1 {
+    font-size: 2.5rem;
+    font-weight: 800;
+    line-height: 3rem;
+    background-clip: initial;
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+    background-image:
+      -webkit-linear-gradient(
+        45deg,
+        #f40001,
+        #ec5242,
+        rgb(255, 64, 0),
+        rgb(255, 123, 0),
+        #a00000,
+        #f65112,
+        #ec5242
+      );
+  }
+
+  .description {
+    width: 68%;
+    font-weight: 400;
+    font-size: 1.3rem;
+    line-height: 1.6rem;
+  }
+
+  .date {
+    font-size: 2.7rem;
+  }
+
+  .location {
+    font-size: 1.5rem;
+  }
+
+  .all-programs {
+    display: flex;
+    gap: 0.6rem;
+  }
+
+  .program-card {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .program-desc {
+    text-align: center;
+  }
+
+  .join-btn {
+    display: none;
+  }
+
+  .join-btn2 {
+    display: block;
+    font-family: 'Lato', sans-serif;
+    font-weight: 400;
+    font-size: 1rem;
+    background: transparent;
+    color: #fff;
+    padding: 1.25rem;
+    margin-top: 1rem;
+    text-decoration: underline;
+  }
+
+  .all-speakers {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 0.5rem;
+  }
+
```

Comment:
- [OPTIONAL] I think it would be better to add some padding to the top. What do you think? ✨

Your design | Design guidelines
---| ---|
![image](https://user-images.githubusercontent.com/106119105/235274577-51a722a5-705f-4a04-9144-c76650f5631e.png) | ![image](https://user-images.githubusercontent.com/106119105/235274602-1bc1a9e4-1f60-46ea-851f-2124428ac6cd.png)